Source/ISHRemote/Trisoft.ISHRemote/Cmdlets/DocumentObj/SetIshDocumentObj.Tests.ps1

```diff
@@ -0,0 +1,759 @@
+			$ishObjectB = Add-IshDocumentObj -IshSession $ishSession -IshFolder $ishFolderTopic -IshType ISHModule -Lng $ishLng -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent
+			
+			$updatedTitle = $ishObjectA.ftitle_logical_value + "...updated"								
+			$ishMetadataFieldsSet = Set-IshMetadataField -IshSession $ishSession -Name "FTITLE" -Level "Logical" -Value $updatedTitle
+			
+			$ishObjectArray = Set-IshDocumentObj -IshSession $ishSession -IshObject @($ishObjectA, $ishObjectB) -Metadata $ishMetadataFieldsSet
+			
+			[xml]$xmlRevisionInfoA = $ishSession.DocumentObj25.RetrieveRevisionInfoByIshLngRefs(@($ishObjectA.LngRef), "")
+			[xml]$xmlRevisionInfoB = $ishSession.DocumentObj25.RetrieveRevisionInfoByIshLngRefs(@($ishObjectB.LngRef), "")
+
+			$ishObjectArray.Count | Should Be 2
+			$xmlRevisionInfoA.SelectNodes("//ishobjects/ishobject/ishchanges/ishchange").Count | Should Be 1
+			$xmlRevisionInfoB.SelectNodes("//ishobjects/ishobject/ishchanges/ishchange").Count | Should Be 1
+			
+			$ishObjectArray[0].ftitle_logical_value | Should Be $updatedTitle
+			$ishObjectArray[1].ftitle_logical_value | Should Be $updatedTitle
+		}
+		
+		It "Resubmit blob, provide multiple objects to IshObject" {
+			$ishObjectA = Add-IshDocumentObj -IshSession $ishSession -IshFolder $ishFolderTopic -IshType ISHModule -Lng $ishLng -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent | Get-IshDocumentObjData
+			$ishObjectB = Add-IshDocumentObj -IshSession $ishSession -IshFolder $ishFolderTopic -IshType ISHModule -Lng $ishLng -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent | Get-IshDocumentObjData
+			
+			$ishObjectArray = Set-IshDocumentObj -IshSession $ishSession -IshObject @($ishObjectA, $ishObjectB)
+			[xml]$xmlRevisionInfoA = $ishSession.DocumentObj25.RetrieveRevisionInfoByIshLngRefs(@($ishObjectA.LngRef), "")
+			[xml]$xmlRevisionInfoB = $ishSession.DocumentObj25.RetrieveRevisionInfoByIshLngRefs(@($ishObjectB.LngRef), "")
+
+			$ishObjectArray.Count | Should Be 2
+			$xmlRevisionInfoA.SelectNodes("//ishobjects/ishobject/ishchanges/ishchange").Count | Should Be 2
+			$xmlRevisionInfoB.SelectNodes("//ishobjects/ishobject/ishchanges/ishchange").Count | Should Be 2
+		}
+		
+		It "Resubmit blob, provide multiple objects via pipeline" {
+			$ishObjectA = Add-IshDocumentObj -IshSession $ishSession -IshFolder $ishFolderTopic -IshType ISHModule -Lng $ishLng -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent
+			$ishObjectB = Add-IshDocumentObj -IshSession $ishSession -IshFolder $ishFolderTopic -IshType ISHModule -Lng $ishLng -Metadata $ishTopicMetadata -FileContent $ditaTopicFileContent
+			$ishObjectAB = @($ishObjectA, $ishObjectB) | Get-IshDocumentObj -IshSession $ishSession -IncludeData
+			
+			$ishObjectArray = $ishObjectAB | Set-IshDocumentObj -IshSession $ishSession
+			[xml]$xmlRevisionInfoA = $ishSession.DocumentObj25.RetrieveRevisionInfoByIshLngRefs(@($ishObjectA.LngRef), "")
```

Comment:
Right, but with checking number of revisions I am testing if the blob was submitted by Set-IshDocumentObj or not. Since we have variations in DocumentObj25 api (Update, SetMetadata) I want to make sure the proper code path was used in Set-IshDocumentObj.

Reply:
Will try to simplify using  Get-IshDocumentObj